### PR TITLE
Avtar

### PR DESCRIPTION
**Pull request by Varun-28**
Done with avatar component, it is of three types - simple, sizes, square.
Review It !
avatar1
avatar2
avatar3

### CODE REVIEW

File: css/component_styles/avatar.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+/* profile avtar */
+img.avatar {
+    width: 3rem;
+    height: 3rem;
+    border-radius: 50%;
+    margin: auto 1rem;
+}
+
+#avatar1{
```
Comment (raghav2110) [inline css/component_styles/avatar.css]: Can use better naming .More generic i suppose.

File: css/component_styles/avatar.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+/* sizes avtar */
+img.square{
+    border-radius: 5px;
+}
+
+#avatar2, #avatar3{
+    min-height: 15rem;
+}
```
Comment (Hamdrive) [inline css/component_styles/avatar.css:34]: Add extra empty line to fix EOF issue

**Review by Hamdrive — COMMENTED** [review]
Overall looks good. Component's look good.

File: css/component_styles/avatar.css (hunk 2 above)
Comment (Varun-28) [reply css/component_styles/avatar.css:34]: done 👍✔️🛠️

**Pull request merged**